docs/src/components/SDKCards.astro

```diff
@@ -63,15 +64,26 @@ const sdks = [
     linkUrl: "https://pypi.org/project/ultravox-client/",
     repo: "https://github.com/fixie-ai/ultravox-client-sdk-python",
   },
+  {
+    title: "iOS/Swift",
+    logo: swiftLogo,
+    logoAlt: "Swift logo",
+    introLinkText: "Find it on ",
+    linkText: "Swift Package Index",
+    linkUrl: "https://swiftpackageindex.com/fixie-ai/ultravox-client-sdk-ios",
+    repo: "https://github.com/fixie-ai/ultravox-client-sdk-ios",
+  },
 ];
+
+const sortedSDKs = sdks.sort((a, b) => a.title.localeCompare(b.title));
```

Comment:
Up to you, but I'd consider manually sorting these instead of automatically alphabetizing them because e.g. we may want to steer people toward the platforms that are easier for us to support and that will consequently get updates faster.

Reply:
yeah good idea